pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -53,11 +57,19 @@
 from pytorch_lightning.utilities.exceptions import DeadlockDetectedException, MisconfigurationException
 from pytorch_lightning.utilities.seed import reset_seed
 
+if not _IS_WINDOWS and _TORCH_GREATER_EQUAL_1_9:
+    from torch.distributed.optim import DistributedOptimizer, PostLocalSGDOptimizer, ZeroRedundancyOptimizer
+
+if _FAIRSCALE_AVAILABLE:
+    from fairscale.optim import OSS
 if _HYDRA_AVAILABLE:
     from hydra.core.hydra_config import HydraConfig
     from hydra.utils import get_original_cwd, to_absolute_path
 if _TORCH_GREATER_EQUAL_1_8:
     from pytorch_lightning.utilities.distributed import register_ddp_comm_hook
+if _TORCH_GREATER_EQUAL_1_9:
+    import torch.distributed.algorithms.ddp_comm_hooks.post_localSGD_hook as post_localSGD
+    import torch.distributed.algorithms.model_averaging.averagers as averagers
```

Comment:
These should be merged with the `if not _IS_WINDOWS and _TORCH_GREATER_EQUAL_1_9` import check above, as they are always used under those conditions

Reply:
`not _IS_WINDOWS` is no longer a constraint here. It's fixed this week. Will update this soon.